app/models/org.rb

```diff
@@ -0,0 +1,20 @@
+class Org < ApplicationRecord
```

Comment:
i think this should also include the security event module, and we should start off by emitting all the relevant security events rather than adding them in follow-ups

Reply:
Copying events from user, the only applicable event seems to be "created", so I have added that.

Reply:
also memberships added/confirmed/removed